tools/convert/templates/rules_docrepo.tmpl

```diff
@@ -7,7 +7,8 @@ It was automatically generated {{ now }}.
 ## The Rules file
 
 The rules file is a YAML file.
-Here is a simple example of a rules file:
+Below is a simple example of a `rules` file.
+View a [complete example](https://github.com/honeycombio/refinery/blob/main/rules_complete.yaml).
```

Comment:
@kentquirk Not sure if the templates can support markdown format for links or not... Is this possible or what should be used instead?